### DISCUSSION TITLE
MOOSE update error - undefined reference

### DISCUSSION THREAD

**Discussion by LukeFisherUKAEA** [idaholab/moose#23191] [Q&A Getting Started] [answered]
After a standard update of MOOSE using the commands in the [Update MOOSE and Conda page](https://mooseframework.inl.gov/getting_started/installation/update_moose.html), I am receiving the following error during execution of the _make_ command.  I have checked and the 'libtiff' package is installed in the moose environment.  Any help would be much appreciated.  Thanks.
![libtiff_error](https://user-images.githubusercontent.com/91187320/213220232-c8733ec9-4e3f-4b7a-a22c-aaf94f483d62.PNG)
![libtiff_error_2](https://user-images.githubusercontent.com/91187320/213220239-4cb1b3c2-b9b1-438b-a748-25fdd692f6ff.PNG)

**Comment by GiudGiud** [comment, marked as answer]
Hello

This is odd, it comes from a dependency not really from us. 
I d consider wiping the environment and installing it again. It's likely easier than debugging it. 

Guillaume

**Reply by LukeFisherUKAEA** [reply]
Hello Guillaume,

Many thanks for this.  I'll go ahead with a new installation.

Luke
